src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
```

Comment:
Would be better to use an enum for Color and Shape.

Reply:
totally. Convert to enum and adjust using enum in the following code.